README.md

```diff
@@ -69,44 +68,94 @@ Required options:
 
 ### Working with Images
 
-* `hokusai build` - Build the latest docker image for the project.
-* `hokusai pull` - Pull images for your project from your AWS ECR repo.
-* `hokusai push` - Push a locally built image to your AWS ECR repo.
-* `hokusai images` - List all project images in your local docker registry.
+* `hokusai push` - Build and push an image to the AWS ECR project repo, by default tagged as the output of `git rev-parse HEAD`.
+* `hokusai images` - List image metadata in the AWS ECR project repo.
 
 ### Working with Kubernetes
-Hokusai uses `kubectl` to connect to Kubernetes. You first need to make sure `kubectl` is installed and you have proper config setup for connecting to your Kubernetes. Hokusai `install` commands provide basic setup for this:
+Hokusai uses `kubectl` to connect to Kubernetes. You first need to make sure `kubectl` is installed and you have proper config setup for connecting to your Kubernetes. Hokusai `deps` commands provide basic setup for this:
 ```bash
-hokusai install --help
+hokusai deps --help
 ```
 Recommended approach is to upload your `kubectl` config to S3 and use following command to install it:
 ```bash
-hokusai install --s3-bucket <bucket name> --s3-key <file key>
+hokusai deps --s3-bucket <bucket name> --s3-key <file key>
 ```
-### Working with ConfigMaps
-
-* `hokusai config pull` - Pulls config from the Kubernetes server and writes to the `hokusai` directory.
-* `hokusai config push` - Pushes config from the hokusai directory to the Kubernetes server. Config is created for the project as the Kubernetes ConfigMap object `{project}-config`
 
+When running `hokusai setup` `staging.yml` and `production.yml` are created in the hokusai project directory. These files define what Hokusai calls "stacks", and Hokusai is opinionated about a workflow between a staging and production Kubernetes context.  Hokusai commands such as `stack`, `secrets`, `deploy` and `logs` require invocation with either the `--staging` or `--production` flag, which references the respective stack YAML file and interacts with the respective Kubernetes context.
```

Comment:
Having `--staging` and `--production` as flags themselves rather than options (like `-c staging` or something) seems arbitrary to me. We should do whatever's typical in the pip/cli world I guess.